components/formats-bsd/src/loci/formats/tiff/TiffParser.java

```diff
@@ -1026,7 +1026,9 @@ else if (stripByteCounts[countIndex] < 0 && countIndex > 0) {
           // we only want a piece of the tile, so read each row separately
           // this is especially necessary for large single-tile images
           int bpp = bytes * effectiveChannels;
-          in.skipBytes((int) (y * bpp * tileWidth));
+          // don't use skipBytes here, as the number of bytes to skip may
+          // be greater than Integer.MAX_VALUE if the tile is large
```

Comment:
Could just use `skipBytes(long)`.

Reply:
Forgot about that API method; fixed in b47dafa.